bridge/launchpad/import.go

```diff
@@ -70,6 +73,47 @@ func (li *launchpadImporter) ImportAll(repo *cache.RepoCache) error {
 			fmt.Println("TODO: Update bug")
 		}
 
+		/* Handle messages */
+		if len(lpBug.Messages) == 0 {
+			return errors.Wrapf(err, "failed to fetch comments for bug #%s", lpBugID)
+		}
+
+		// The Launchpad API returns the bug description as the first
+		// comment, so skip it.
+		for i, lpMessage := range lpBug.Messages[1:] {
+			commentID := fmt.Sprintf("%d-%d", lpBug.ID, i)
```

Comment:
Are you sure that the list of comments you get from Launchpad is stable ? That is, can a comment be deleted and if yes, does that change the list ? Because, as you use through `i` the position of the comment in the response, that could break the mapping. Maybe use `self_link` that seems more stable (taken from https://launchpad.net/+apidoc/1.0.html#message, i suppose that the correct one) ?

Reply:
I did it.